src/content/components/IntervalMarkerOverview.js

```diff
@@ -1,44 +1,189 @@
 import React, { Component, PropTypes } from 'react';
 import shallowCompare from 'react-addons-shallow-compare';
+import classNames from 'classnames';
+import { timeCode } from '../../common/time-code';
 import { withSize } from '../with-size';
 
+type MarkerState = 'PRESSED' | 'HOVERED' | 'NONE';
```

Comment:
At some point I'll stop gushing about flow, but I love this.

Also, flow won't be checked unless you have.`// @flow` at the top of flow files.